inductiva/__init__.py

```diff
@@ -18,6 +18,7 @@
 from . import utils
 from . import stellarators
 from . import world
+from inductiva.tasks import Task
```

Comment:
What is the reason to have this import here?

Reply:
I removed it now